genai-perf/genai_perf/wrapper.py

```diff
@@ -113,17 +113,18 @@ def build_cmd(args: Namespace, extra_args: Optional[List[str]] = None) -> List[s
 
         utils.remove_file(args.profile_export_file)
 
+        # TODO: Temp, remove below change
         cmd = [
-            f"perf_analyzer",
+            f"build/install/bin/perf_analyzer",
             f"-m",
             f"{args.formatted_model_name}",
```

Comment:
since we ripped out `--async` from this default command list, can we also rip the `-m` out as well? Since dynamic grpc doesn't use this flag.

Reply:
Sure, I'll add this. We still require a model be passed in anyway though, so it won't make much of a difference. If we want to remove the model name being required, that's a bigger change. I think we use it in so many places (artifacts, visualizations, etc.) that we'd want its own PR for making model name optional. I don't think it's worth the effort unless it becomes optional for multiple use cases.